addon/session-stores/cookie.js

```diff
@@ -143,8 +166,7 @@ export default BaseStore.extend({
   },
 
   _read(name) {
-    let value = document.cookie.match(new RegExp(`${name}=([^;]+)`)) || [];
-    return decodeURIComponent(value[1] || '');
+    return this.get('_cookies').read(name) || '';
```

Comment:
I'm going to fix that. Sorry for that - I really shouldn't be breaking great work of people who want to contribute 😔